files/restore_backup.py

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env python
```

Comment:
This module is also missing the standard GPLv3 license header normally found in modules.